lms/envs/common.py

```diff
@@ -4167,6 +4167,8 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
         'account_privacy',
         'profile_image',
         'username',
+        'hide_username',
+        'name',
```

Comment:
I don't think we should make name visible for all by default. Lets find an alternative for this (maybe use edx-internal?)